infrastructure/joke_crawler.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_lambda_function" "joke_crawler" {
+  function_name = "joke_crawler"
+  s3_bucket     = "joke-src-bucket-dzaa1417"
+  s3_key        = "joke-crawler.zip"
+  role          = "arn:aws:iam::208631683274:role/LabRole"
```

Comment:
I would use the terraform provided account id. 
Maybe define a local variable to create the role arn